docs/spec-conformance.md

```diff
@@ -0,0 +1,17 @@
+# Spec conformance
+
+This branch of runc implements the [OCI Runtime Spec v1.1.0-rc.1](https://github.com/opencontainers/runtime-spec/tree/v1.1.0-rc.1)
+for the `linux` platform.
+
+The following features are not implemented yet:
+
+Spec version | Feature                                  | PR
+-------------|------------------------------------------|----------------------------------------------------------
+v1.0.0       | `SCMP_ARCH_PARISC`                       | Unplanned, due to lack of users
+v1.0.0       | `SCMP_ARCH_PARISC64`                     | Unplanned, due to lack of users
+v1.0.2       | `.linux.personality`                     | [#3126](https://github.com/opencontainers/runc/pull/3126)
+v1.1.0-rc.1  | `.linux.resources.cpu.burst`             | [#3205](https://github.com/opencontainers/runc/pull/3205)
+v1.1.0-rc.1  | `.[]mounts.uidMappings`                  | [#3429](https://github.com/opencontainers/runc/pull/3429)
+v1.1.0-rc.1  | `.[]mounts.gidMappings`                  | [#3429](https://github.com/opencontainers/runc/pull/3429)
```

Comment:
As I mentioned in that PR, I think that approach can't work (can't create an idmap mount within a userns). I opened another PR that works: https://github.com/opencontainers/runc/pull/3717

Can we mention #3717 here?

Reply:
updated